Makefile

```diff
@@ -24,7 +24,7 @@ testhttpbin:
 	cd test-it && node test.js -j -t httpbin --dots
 
 testjs:
-	cd test-it && node test.js -j --dots
+	cd test-it && node test.js -j 
```

Comment:
Just for consistency, please make the same change to testnative case. If I remember correctly in failure conditions having the dots is not really helpful.

Reply:
This was mistakenly submitted.  It was for debugging a CI failure and I forgot to back it out.  Retracted.

Reply:
Although I agree the dots are not helpful, Milan really like them though.  If you agree they should be removed I wouldn't be averse to trying to win over Milan.